x/pricefeed/simulation/operations/msg.go

```diff
@@ -98,6 +83,51 @@ func SimulateMsgUpdatePrices(keeper keeper.Keeper) simulation.Operation {
 	}
 }
 
+func getStartPrice(marketID string) (startPrice sdk.Dec) {
+	switch marketID {
+	case "btc:usd":
+		return sdk.MustNewDecFromStr("7000")
+	case "bnb:usd":
+		return sdk.MustNewDecFromStr("15")
+	case "xrp:usd":
+		return sdk.MustNewDecFromStr("0.25")
+	}
+	return sdk.MustNewDecFromStr("100")
+}
+
+func getIncrement(marketID string) (increment sdk.Dec) {
+	startPrice := getStartPrice(marketID)
+	divisor := sdk.MustNewDecFromStr("20")
+	increment = startPrice.Quo(divisor)
+	return increment
+}
+
+func setPrice(marketID string, price sdk.Dec) {
+	switch marketID {
+	case "btc:usd":
+		btcPrices = append(btcPrices, price)
+		return
+	case "bnb:usd":
+		bnbPrices = append(bnbPrices, price)
+		return
+	case "xrp:usd":
+		xrpPrices = append(xrpPrices, price)
+	}
+	return
+}
+
+func pickNewRandomPrice(marketID string, blockHeight int) (newPrice sdk.Dec) {
+	switch marketID {
+	case "btc:usd":
+		return btcPrices[blockHeight-1]
+	case "bnb:usd":
+		return bnbPrices[blockHeight-1]
+	case "xrp:usd":
+		return xrpPrices[blockHeight-1]
+	}
+	panic("invalid price request")
```

Comment:
Ah yes good to include panic here instead of just returning a default price as it was before